man/man8/ipa-healthcheck.8

```diff
@@ -70,28 +70,28 @@ Take as input a JSON results output and convert it to a more human\-readable for
 .PP
 Execute healthcheck with the default JSON output:
 .PP
-.Vb 1
-\& ipa\-healthcheck
-.Ve
+.nf 1
+\&# ipa\-healthcheck
+.fi
 .PP
 Execute healthcheck with a prettier JSON output:
 .PP
-.Vb 1
-\& ipa\-healthcheck \-\-indent 2
-.Ve
+.nf 1
+\&# ipa\-healthcheck \-\-indent 2
+.fi
 .PP
 Execute healthcheck and only display errors:
 .PP
-.Vb 1
-\& ipa\-healthcheck \-\-failures\-only
-.Ve
+.nf 1
+\&# ipa\-healthcheck \-\-failures\-only
+.fi
 .PP
 Display in human\-readable output a previous report:
 .PP
-.Vb 2
-\& ipa\-healthcheck \-\-output\-format human \-\-input\-file \e
+.nf 2
+\&# ipa\-healthcheck \-\-output\-type human \-\-input\-file \e
 \&        /var/log/ipa/check.json
```

Comment:
Can you change one more thing, the log location in the sample to /var/log/ipa/healthcheck/healthcheck.log ? This is the default location that the systemd timer logs to

Reply:
Done.